cirq/google/engine/engine.py

```diff
@@ -694,8 +700,25 @@ def get_calibration(self, calibration_name: str) -> 'Calibration':
         return Calibration(response['data']['data'])
 
 
-class Calibration:
-    """A convenience wrapper for calibrations
+class Calibration(abc.Mapping):
+    """A convenience wrapper for calibrations that acts like a dictionary.
+
+    Calibrations act as dictionaries whose keys are the names of the metric,
+    and whose values are the metric values.  The metric values themselves are
+    represented as a dictionary.  These metric value dictionaries have
+    keys that are tuples of `cirq.GridQubit`s and values that are lists of the
```

Comment:
What do you think of using a frozenset of GridQubits for the key instead of a tuple since order shouldn't change the match?

Reply:
I'm not sure that the order won't matter.  For example if you have a directional gate (like a cnot that goes from a to b), then you might have metrics about that gate whose order differs (a to b versus b to a).